Anonymous author short use case Will Fix #60

philarcher opened the pull request:
See Issue #60 for information


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/pull/86.html" title="Last updated on Jun 30, 2020, 9:30 AM UTC (1b304dd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/86/702b794...1b304dd.html" title="Last updated on Jun 30, 2020, 9:30 AM UTC (1b304dd)">Diff</a>

philarcher committed:
Anonymous author short use case Will Fix #60

jandrieu commented on index.html:
```diff
@@ -378,6 +378,16 @@ <h3>Sharing opted-in information across platforms</h3>
+	industries: she was a ghostwriter, a scriptfixer, various kinds of editor, a translator, and more. Some of these gigs 
+	still pay her residuals years later in the form of tiny micropayments. When these now-classic works are reprinted or 
+	syndicated, the payments are automatically sent out, without anyone involved thinking much about who exactly the contract 
+	refers to as “anonymous ghostwriter X” or “translator of appendix C”. Using a DID as her identifier, rather than her name, allows 
```
How about "the payments are automatically sent using information associated with Sahar's DID"?

philarcher committed:
Including Joe's suggestion

Joe's suggestion of "the payments are automatically sent using information associated with Sahar's DID" has been included

philarcher committed:
Merge branch 'master' into philarcher-patch-18
